### PR TITLE
Deploy 2024.08.01

### PR DESCRIPTION
**Pull request by siers**

### CODE REVIEW

File: release.mk [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # for DSP, API, APP, see https://github.com/dasch-swiss/ops-deploy/blob/main/versions/RELEASE.json
 # for TOOLS, see https://github.com/dasch-swiss/dsp-tools/releases
-DSP := "2024.07.02"
-API := "v30.18.2"
-APP := "v11.16.3"
+DSP := "2024.08.01"
+API := "v30.18.3"
+APP := "v11.17.1"
 TOOLS := "v8.4.0"
```
Comment (jnussbaum) [inline release.mk:6]: Before deploying the docs, we need a release of DSP-TOOLS that can be entered here.

**Review by jnussbaum — CHANGES_REQUESTED** [review]

File: release.mk (hunk 1 above)
Comment (siers) [reply release.mk]: ✔️

Comment (jnussbaum) [reply release.mk]: The submodule of dsp-tools needs to be updated, too

Comment (siers) [reply release.mk]: ✔️

**Review by jnussbaum — APPROVED** [review]

**Pull request merged**